third_party/pyth/multisig-wh-message-builder/src/index.ts

```diff
@@ -0,0 +1,284 @@
+import {
+  importCoreWasm,
+  ixFromRust,
+  setDefaultWasm,
+  utils as wormholeUtils,
+} from "@certusone/wormhole-sdk";
+import * as anchor from "@project-serum/anchor";
+import NodeWallet from "@project-serum/anchor/dist/cjs/nodewallet";
+import {
+  Keypair,
+  LAMPORTS_PER_SOL,
+  PublicKey,
+  SystemProgram,
+} from "@solana/web3.js";
+import Squads from "@sqds/sdk";
+import bs58 from "bs58";
+import { program } from "commander";
+import * as fs from "fs";
+
+setDefaultWasm("node");
+
+program
+  .name("pyth-multisig")
+  .description("CLI to creating and executing multisig transactions for pyth")
+  .version("0.1.0");
+
+program
+  .command("create")
+  .description("Create a new multisig transaction")
+  .option("-c, --cluster <network>", "solana cluster to use", "devnet")
+  .option("-v, --vault-address <address>", "multisig vault address")
+  .option(
+    "-w, --wallet <filepath>",
+    "multisig wallet secret key filepath",
+    "keys/key.json"
+  )
+  .option("-p, --payload <string>", "payload to sign", "hello world")
+  .action((options) => {
+    createMultisigTx(
+      options.cluster,
+      new PublicKey(options.vaultAddress),
+      options.wallet,
+      options.payload
+    );
+  });
+
+program
+  .command("execute")
+  .description("Execute a multisig transaction that is ready")
+  .option("-c, --cluster <network>", "solana cluster to use", "devnet")
+  .option("-e, --emitter-address <address>", "emitter address")
+  .option(
+    "-w, --wallet <filepath>",
+    "multisig wallet secret key filepath",
+    "keys/key.json"
+  )
+  .option(
+    "-m, --message <filepath>",
+    "multisig message account secret key filepath",
+    "keys/message.json"
+  )
+  .option("-t, --tx-pda <address>", "transaction PDA")
+  .option("-u, --rpc-url <url>", "wormhole RPC URL")
```

Comment:
we could -- the reason I made it an arg was for flexibility in case one rpc url goes down and we can easily switch to another one, happy to hardcode tho if that doesn't seem much of an issue

Reply:
Makes sense to me. let's leave it as it is. just maybe it is worth to give a link to this page https://book.wormhole.com/reference/rpcnodes.html in readme.